Dockerfile-apigw.static

```diff
@@ -0,0 +1,10 @@
+FROM dojot/kong:v0.2.1
+
+RUN yum -y update && yum -y install git && mkdir /plugins
+
+RUN cd /plugins; git clone https://github.com/dojot/ma-kong-plugin.git mutualauthentication
```

Comment:
Shouldn't we remove mutual-authentication from dojot once it needs to be tested?

Reply:
I just wanted to mimic what it was before.